docs/install.md

```diff
@@ -14,15 +23,20 @@ To run Sugarizer Server using Docker and Docker Compose:
 
 Install Docker Compose
 
-	curl -L "https://github.com/docker/compose/releases/download/1.8.1/docker-compose-$(uname -s)-$(uname -m)" > /usr/local/bin/docker-compose
+	curl -L "https://github.com/docker/compose/releases/download/1.8.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
 	chmod +x /usr/local/bin/docker-compose
 
+If you have problems installing with `curl`, you can use `pip` instead: `pip install -U docker-compose`
```

Comment:
There are certain conditions while installing docker-compose with pip which needs to be fulfilled.
See: https://docs.docker.com/compose/install/#alternative-install-options

It's misleading to suggest pip as the installation method without highlighting the recommended requirements.

For example, the users are not aware that "If you install using pip, we recommend that you use a [virtualenv](https://virtualenv.pypa.io/en/latest/) because many operating systems have python system packages that conflict with docker-compose dependencies."

So, instead of suggesting the use of pip to install docker compose, it's better to add a reference to this (https://docs.docker.com/compose/install/) documentation so that user can install correct version of docker-compose based on their OS.